lib/peer.js

```diff
@@ -1,60 +1,79 @@
 'use strict';
 
-var Buffers = require('buffers');
+var Buffers = require('./buffers');
 var EventEmitter = require('events').EventEmitter;
 var Net = require('net');
 var Socks5Client = require('socks5-client');
-var util = require('util');
-
 var bitcore = require('bitcore');
-var $ = bitcore.util.preconditions;
 var Networks = bitcore.Networks;
 var Messages = require('./messages');
-
-var MAX_RECEIVE_BUFFER = 10000000;
+var $ = bitcore.util.preconditions;
+var util = require('util');
 
 /**
- * A Peer instance represents a remote bitcoin node and allows to communicate
- * with it using the standard messages of the bitcoin p2p protocol.
+ * A constructor to create Peer instances to send and recieve messages
+ * using the standard Bitcoin protocol.
  *
```

Comment:
You're right that constructor is better here as you had it. There's a typo though *receive. In retrospect my suggestion was really as an update to docs/peer.md class description.